# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/sysstat/prefix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -dur sysstat-6.0.1/build/CONFIG sysstat-6.0.1-p/build/CONFIG
--- sysstat-6.0.1/build/CONFIG	2005-06-25 11:14:21.000000000 +0200
+++ sysstat-6.0.1-p/build/CONFIG	2005-10-23 22:55:42.000000000 +0200
@@ -2,8 +2,8 @@
 # (C) 2000 Sebastien GODARD (sysstat <at> wanadoo.fr)
 
 # Directories
-PREFIX = /usr/local
-SA_LIB_DIR = /usr/local/lib/sa
+PREFIX = /usr
+SA_LIB_DIR = /usr/lib/sa
 SADC_PATH = ${SA_LIB_DIR}/sadc
 SA_DIR = /var/log/sa
 MAN_DIR = ${PREFIX}/man
diff -dur sysstat-6.0.1/build/CONFIG.def sysstat-6.0.1-p/build/CONFIG.def
--- sysstat-6.0.1/build/CONFIG.def	2004-12-30 19:53:06.000000000 +0100
+++ sysstat-6.0.1-p/build/CONFIG.def	2005-10-23 22:54:51.000000000 +0200
@@ -2,8 +2,8 @@
 # (C) 2000 Sebastien GODARD (sysstat <at> wanadoo.fr)
 
 # Directories
-PREFIX = /usr/local
-SA_LIB_DIR = /usr/local/lib/sa
+PREFIX = /usr
+SA_LIB_DIR = /usr/lib/sa
 SADC_PATH = ${SA_LIB_DIR}/sadc
 SA_DIR = /var/log/sa
 MAN_DIR = ${PREFIX}/man
diff -dur sysstat-6.0.1/build/Configure.sh sysstat-6.0.1-p/build/Configure.sh
--- sysstat-6.0.1/build/Configure.sh	2004-10-10 16:55:49.000000000 +0200
+++ sysstat-6.0.1-p/build/Configure.sh	2005-10-23 22:53:54.000000000 +0200
@@ -10,10 +10,10 @@
 
 # Installation directory
 
-PREFIX=`${ASK} 'Installation directory:' "/usr/local" "prefix"`
+PREFIX=`${ASK} 'Installation directory:' "/usr" "prefix"`
 if [ ! -d ${PREFIX} ]; then
-	echo "WARNING: Directory ${PREFIX} not found: Using default (/usr/local)."
-	PREFIX=/usr/local
+	echo "WARNING: Directory ${PREFIX} not found: Using default (/usr)."
+	PREFIX=/usr
 fi
 
 # sadc directory
